### DISCUSSION TITLE
GET Data as JSON File

### DISCUSSION THREAD

**Discussion by SplendidMinds** [jomjol/AI-on-the-edge-device#829] [General]
Hi - is it possible to just request the current Data as a JSON File? Like http://watercounter/getdata and I receive a JSON. This would be a big benefit. Thank you very much.

**Comment by jomjol** [comment]
This is already implemented: http://ip-adress/json

**Comment by tutu81** [comment]
@jomjol : was searching this info in the wiki (feature). Is it documented or should I add it ? 
may be it could make sense to have a wiki page with "expose data" => MQTT, JSON, Influx etc.

**Comment by jomjol** [comment]
Good idea!
